src/System.ComponentModel.TypeConverter/tests/Drawing/ImageConverterTests.cs

```diff
@@ -0,0 +1,228 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Globalization;
+using System.IO;
+using Xunit;
+
+namespace System.ComponentModel.TypeConverterTests
+{
+    public class ImageConverterTest
+    {
+        private readonly Image _image;
+        private readonly ImageConverter _imgConv;
+        private readonly ImageConverter _imgConvFrmTD;
+        private readonly string _imageStr;
+        private readonly byte[] _imageBytes;
+
+        public ImageConverterTest()
+        {
+            var assembly = System.Reflection.Assembly.GetExecutingAssembly();
+
+            var imageName = "TestImage.bmp";
+            Stream testImageStream = assembly.GetManifestResourceStream(imageName);
+            if (testImageStream == null)
+            {
+                throw new InvalidOperationException($"Resource image \"{imageName}\" not found.");
+            }
+
+            int length = (int)testImageStream.Length;
+            _imageBytes = new byte[length];
+            if (testImageStream.Read(_imageBytes, 0, length) != length)
```

Comment:
I don't think this is necessary, just leave it to Image.FromStream, it will read the stream accordingly and create the Image from it. Remove this logic to validate that the resource was loaded correctly and leave it to Image.FromStream that already does that.